### PR TITLE
Controller scaffolding

### PR DESCRIPTION
**Pull request by embano1**
Basic controller scaffolding w/out resources

### CODE REVIEW

**Review by A-Hilaly — APPROVED** [review]

**Comment by A-Hilaly** [comment]
We need to add @ack-bot as a collaborator to be able to run e2e tests..

**Comment by A-Hilaly** [comment]
From earth to prow, do you copy?
/test all

File: apis/v1alpha1/generator.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ignore:
```
Comment (RedbackThomson) [inline apis/v1alpha1/generator.yaml:1]: A controller without any resources breaks some of our CI/CD, but I can't remember exactly where. I'll have to keep an eye on this as it goes through

**Review by RedbackThomson — CHANGES_REQUESTED** [review]
Could you please edit the `metadata.yaml` file with the up-to-date information. If we onboard a controller without that metadata, it will break the documentation website and Helm chart

File: apis/v1alpha1/generator.yaml (hunk 1 above)
Comment (embano1) [reply apis/v1alpha1/generator.yaml:1]: This was per @A-Hilaly request to exclude all resources for the scaffolding PR and add each resource afterwards. I have a fully working `generator.yaml` in [my branch](https://github.com/embano1/eventbridge-controller/blob/aminemicha/generator.yaml), not sure which fields you want me to add to this base?

Comment (embano1) [reply apis/v1alpha1/generator.yaml:1]: Sorry, my bad and misunderstanding. Updated the file

**Comment by embano1** [comment]
> Could you please edit the `metadata.yaml` file with the up-to-date information. If we onboard a controller without that metadata, it will break the documentation website and Helm chart

Updated the `metadata.yaml`

**Comment by A-Hilaly** [comment]
Manually merging as tests will not pass until we introduce crds.
Thank you Michael!

**Pull request merged**